### DISCUSSION TITLE
Section ordering

### DISCUSSION THREAD

**Discussion by llegirdonagustet** [mateusz-bak/openreads#771] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I use the ordening filter on Finished books I like to order by Finish date but when I use it on Reading section I'd like to order books by Started date. There is only one criteria to order all sections.

**Describe the solution you'd like**
The possibility to order each section by different criteria.

**Comment by OrionCrim** [comment]
I'd like to add support to this request.
The ability to sort each of the lists independently would be a great addition.

**Comment by mateusz-bak** [comment]
Hi, in the latest release it was added - separate sort and filter for each list
